### DISCUSSION TITLE
yes

### DISCUSSION THREAD

**Discussion by SMOLKEYS** [SMOLKEYS/yellow-java#8] [General]
talk.

**Comment by SMOLKEYS** [comment]
@xzxADIxzx can i get a bit of help

**Reply by xzxADIxzx** [reply]
Of course, what do you need?

**Reply by SMOLKEYS** [reply]
> Of course, what do you need?

Can we talk through discord instead of... *here?*

**Reply by SMOLKEYS** [reply]
(SMOLKEYS#4156)

**Comment by SMOLKEYS** [comment]
...what the fuck, its already been 2 days?

**Comment by SMOLKEYS** [comment]
man...
